empty annotations not handled properly

When the pod is terminating, annotations might be empty resulting in 

```
[2019-04-11 09:54:59 +0000]

[7] [ERROR] Error handling request /jupyterhub/services/notebooks/servers
Traceback (most recent call last):
File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base_async.py", line 56, in handle
self.handle_request(listener_name, req, client, addr)
File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/ggevent.py", line 160, in handle_request
addr)
File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base_async.py", line 107, in handle_request
respiter = self.wsgi(environ, resp.start_response)
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2309, in __call__
return self.wsgi_app(environ, start_response)
File "/app/src/notebooks_service.py", line 98, in __call__
return self.app(environ, start_response)
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2295, in wsgi_app
response = self.handle_exception(e)
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1741, in handle_exception
reraise(exc_type, exc_value, tb)
File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
raise value
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
response = self.full_dispatch_request()
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
reraise(exc_type, exc_value, tb)
File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
raise value
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
rv = self.dispatch_request()
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
return self.view_functions[rule.endpoint](**req.view_args)
File "/app/src/notebooks_service.py", line 146, in decorated
return f(user, *args, **kwargs)
File "/app/src/notebooks_service.py", line 247, in user_servers
for (key, value) in pod_annotations.items()
AttributeError: 'NoneType' object has no attribute 'items'
```

--- a/src/notebooks_service.py
+++ b/src/notebooks_service.py
@@ -240,7 +240,7 @@ def user_servers(user):
 
         for server_name, properties in servers.items():
             pod_annotations = annotations.get(
-                properties.get('state', {}).get('pod_name', {})
+                properties.get('state', {}).get('pod_name', {}), {}
             )
             servers[server_name]['annotations'] = {
                 key: value

